### PR TITLE
PD-690: Change host URL in MongoNav

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

- Change host URL from 'stitch.mongodb.com' to 'realm.mongodb.com'
🎟 _Jira ticket:_ [PD-690](https://jira.mongodb.org/browse/PD-690)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### CODE REVIEW

File: packages/mongo-nav/src/data.ts [hunk 1]
```diff
@@ -114,11 +114,21 @@ export const urlFixtures: Required<URLSInterface> = {
   },
 };
 
+export function getProductHost(today = new Date(Date.now())) {
```
Comment (DesignerDave) [inline packages/mongo-nav/src/data.ts:117]: We're only using this in one place for the next week or two. I don't think we need to make `today` configurable.

File: packages/mongo-nav/src/MongoNav.spec.tsx [hunk 2]
```diff
@@ -417,4 +417,27 @@ describe('packages/mongo-nav', () => {
...
+    test('getProductHost function returns stitch', () => {
+      expect(getProductHost()).toBe('stitch');
+    });
+  });
+
+  describe('when the date is the day of MongoDB World', () => {
+    const mdbworld = new Date('June 8, 2020 0:00:00');
+
+    test('getProductHost function returns realm', () => {
+      const productName = getProductHost(mdbworld);
```
Comment (DesignerDave) [inline packages/mongo-nav/src/MongoNav.spec.tsx:431]: Testing this like so doesn't seem to actually test that our implementation is correct. We don't pass any parameters to the function, so we should be testing that the function returns the correct name when nothing is passed.

File: packages/mongo-nav/src/MongoNav.spec.tsx [hunk 3]
```diff
@@ -417,4 +417,27 @@ describe('packages/mongo-nav', () => {
       });
     });
   });
+
+  describe('when the date is before MongoDB World', () => {
+    test('getProductHost function returns stitch', () => {
+      expect(getProductHost()).toBe('stitch');
```
Comment (DesignerDave) [inline packages/mongo-nav/src/MongoNav.spec.tsx:423]: This test will break on the 8th.

File: packages/mongo-nav/src/data.ts (hunk 1 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/data.ts:117]: The reason to do this is to make it testable. Followed the same convention used in ProjectNav

File: packages/mongo-nav/src/MongoNav.spec.tsx (hunk 2 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/MongoNav.spec.tsx:431]: We do that on line 422.

File: packages/mongo-nav/src/MongoNav.spec.tsx (hunk 3 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/MongoNav.spec.tsx]: You're right, I fixed this.

File: packages/mongo-nav/src/data.ts (hunk 1 above)
Comment (DesignerDave) [reply packages/mongo-nav/src/data.ts:117]: I don't think this is an effective test for how we're using the function.

You might consider overwriting the global Date constructor in the test to set it to the desired dates, and using the function with no parameter like it is in the code.

Comment (DesignerDave) [reply packages/mongo-nav/src/data.ts:117]: https://stackoverflow.com/questions/8922145/overriding-the-javascript-date-constructor

**Pull request merged**